src/util/colors.ts

```diff
@@ -0,0 +1,8 @@
+export const Colors = {
```

Comment:
I've done something slightly different in PR #46, introducing a `colors.css` which creates these colours as CSS custom properties at the `:root` level. I prefer that approach because it's CSS, and because it will allow for easier switching of colours for a "dark mode" at a later date.

I think for now we should merge both approaches and reconcile the differences when we revisit this code.